test/pre-installed-wrangler/package.json

```
@@ -0,0 +1,8 @@
+{
+	"name": "wrangler-action-pre-installed-wrangler-test",
+	"license": "MIT",
+	"private": true,
+	"devDependencies": {
+		"wrangler": "^3.28.0"
```

Since this is a test of checking to see if we can use a preinstalled version then it would make sense (with or without a package-lock file) to make this an explicit version.
```suggestion
		"wrangler": "3.28.0"
```

---

Actually how about we do something really crazy and build a fake wrangler into a `node_modules` directory here that responds to the `version` command with a hardcoded version and also supports some fake Wrangler command such as `test-wrangler-action`. If we end up installing a different version over this then running that command would fail.

---

That is truely a wild idea 🤯 
Any ideas on how to do something like this? The only thing that comes to my mind is something like [patch-package](https://www.npmjs.com/package/patch-package).
Would love to know your thoughts.

---

Nothing so clever. I can push a commit for you to consider if you like

---

I pushed a commit but it didn't quite work in the CI as hoped. I can check again later today.

---

Very interesting idea, I understand the path you were suggesting now. I've pushed a minor fix that I think will help its run in CI. Its working locally for me at least when I run `npx wrangler action-test`.